src/components/TreesMap/osmUtil.ts

```diff
@@ -1,10 +1,28 @@
-export const getOSMEditorURL = (nodeId: number): string => {
-  const mapcompleteUrl = 'https://mapcomplete.osm.be/theme';
+export const getOSMEditorURL = ({
+  nodeId,
+  zoom,
+  lat,
+  lon,
+}: {
+  nodeId: number;
+  zoom?: number;
+  lat?: number;
+  lon?: number;
+}): string => {
+  const mapcompleteUrl = 'https://mapcomplete.org';
   const params = new URLSearchParams();
   params.set(
     'userlayout',
-    'https://tordans.github.io/MapComplete-ThemeHelper/OSM-Berlin-Themes/man_made-walter_well-status-checker/theme.json'
+    'https://raw.githubusercontent.com/technologiestiftung/MapComplete-ThemeHelper/024ef9134987e0ead34f261a6270d5572c4d31a4/OSM-Berlin-Themes/man_made-walter_well-status-checker/theme.json'
```

Comment:
It is good to know that you are using a fork now which means we will need to make a PR agains https://github.com/technologiestiftung/MapComplete-ThemeHelper to get updates live.